teuthology/openstack/setup-openstack.sh

```diff
@@ -230,6 +230,7 @@ function setup_pulpito() {
 
     sudo apt-get -qq install -y --force-yes nginx
     local nginx_conf=/etc/nginx/sites-available/default
+    sudo perl -pi -e 's|root /var/www/html|root /usr/share/nginx/html|' $nginx_conf
```

Comment:
so, in Ubuntu 16.04, nginx changed default root to /var/www/html but this was not reflected in /etc/nginx/sites-available/default ? Strange if that file is packaged with nginx